code/controllers/subsystem/ticker.dm

```diff
@@ -612,6 +612,13 @@ SUBSYSTEM_DEF(ticker)
 	if(end_string)
 		end_state = end_string
 
+	var/statspage = CONFIG_GET(string/roundstatsurl)
+	var/gamelogloc = CONFIG_GET(string/gamelogurl)
+	if(statspage)
+		to_chat(world, "<span class='info'>Round statistics and logs can be viewed at [statspage][GLOB.round_id]</span>")
```

Comment:
Should probably explicitly link this because there is desire to kill goonchat linkify except in OOC